prow/oss/cluster/deck_private_deployment.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
```

Comment:
This file still has some resources being applied to `test-pods`.

Reply:
There are 2 of them, for binding deck service account with test-pods RBAC roles, not quite sure why deck need them

Reply:
I see, those are for fetching pod logs. I'm not sure if they are actually needed due to the way we use kubeconfig files, but shouldn't hurt.